test/CreateResourceTest.php

```diff
@@ -6,55 +6,56 @@
 use GuzzleHttp\Handler\MockHandler;
 use GuzzleHttp\HandlerStack;
 use GuzzleHttp\Psr7\Response;
-use Islandora\Chullo\Chullo;
 use Islandora\Chullo\FedoraApi;
 
 class CreateResourceTest extends \PHPUnit_Framework_TestCase
 {
 
     /**
-     * @covers  Islandora\Chullo\Chullo::createResource
      * @covers  Islandora\Chullo\FedoraApi::createResource
      * @uses    GuzzleHttp\Client
      */
     public function testReturnsUriOn201()
     {
-        $mock = new MockHandler([
+        $mock = new MockHandler(
+            [
             new Response(201, ['Location' => "SOME URI"]),
-        ]);
+            ]
+        );
 
         $handler = HandlerStack::create($mock);
         $guzzle = new Client(['handler' => $handler]);
         $api = new FedoraApi($guzzle);
-        $client = new Chullo($api);
 
-        $result = $client->createResource("");
-        $this->assertSame($result, "SOME URI");
+        $result = $api->createResource("");
+        $this->assertEquals($result->getHeaderLine("Location"), "SOME URI");
+        $this->assertEquals(201, $result->getStatusCode(), "Expected a 201 response.");
     }
-
     /**
-     * @covers  Islandora\Chullo\Chullo::createResource
      * @covers  Islandora\Chullo\FedoraApi::createResource
      * @uses    GuzzleHttp\Client
+     *
+     * TODO: Is this useful anymore?
      */
     public function testReturnsNullOtherwise()
     {
-        $mock = new MockHandler([
+        $mock = new MockHandler(
+            [
             new Response(404),
             new Response(409),
-        ]);
+            ]
+        );
 
         $handler = HandlerStack::create($mock);
         $guzzle = new Client(['handler' => $handler]);
         $api = new FedoraApi($guzzle);
-        $client = new Chullo($api);
 
         // 404
-        $result = $client->createResource("");
-        $this->assertNull($result);
+        $result = $api->createResource("");
+        $this->assertEquals(404, $result->getStatusCode());
```

Comment:
As `FedoraApi` just returns the Response then a successful mocked request is as useful as a non-successful one. I think most of the `...returnsNullOtherwise` and `...returnsFalseOtherwise` tests could be dumped with no effect on code coverage.